src/constants/dialogs.ts

```diff
@@ -5,6 +5,7 @@ export enum DialogTypes {
   ExchangeOffline = 'ExchangeOffline',
   FillDetails = 'FillDetails',
   Help = 'Help',
+  Keplr = 'Keplr',
```

Comment:
Could we rename from `Keplr` to `ExternalNavKeplr` to be a little more descriptive